content/en/container-orchestration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Container Orchestration
+status: Completed
+category: Concept
+---
+
+## What it is
+[Container](/container/) orchestration provides the ability to manage and automate the lifecycle of containerized applications in dynamic environments. This includes different aspects that are also pursued by [Cloud Native Technologies](/cloud-native-tech/) and [Cloud Native Apps](/cloud-native-apps/), such as: deployment, (auto)scaling, auto healing and monitoring.
+It functions like a data center operating system (OS), managing [containerized](/containerization/) applications at scale (just like the OS on your laptop that manages your apps).
+
+## Problem it addresses 
+The rise of [Cloud Native Apps](/cloud-native-apps/) and [technologies](/cloud-native-tech/) as well as the fast changing demands of modern businesses introduced a new set of problems. Managing microservices, security and network related things at scale as well as distributed system in general are hard if not impossible to manage manually.   
+
+## How it helps
+Container orchestration, provides a declarative method of automating much of the work, which makes the operational complexity manageable for development and operations, or DevOps. 
+This includes a variety of tasks that software teams must perform in order to manage the lifecycle of a container, such as provisioning, deployment, scaling (both up and down), networking, load balancing, and other activities.
+Orchestrators are also often [self-healing](https://glossary.cncf.io/self-healing/), ensuring the cluster's actual state always matches the operator's desired state.
```

Comment:
```suggestion
```

This sentence does not really add to the understanding of what a container orchestrator is.